ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/layered/LayeredPendingTransactions.java

```diff
@@ -252,6 +257,19 @@ private void logTransactionForReplayDelete(final PendingTransaction pendingTrans
               return rlp.encoded().toHexString();
             })
         .log();
+    LOG.atInfo()
+        .addMarker(INVALID_TX_REMOVED)
+        .addKeyValue("txhash", pendingTransaction::getHash)
+        .addKeyValue("txlog", pendingTransaction::toTraceLog)
```

Comment:
If your concern is about the size of the logged message, then the way I set up the log with the marker and key-value, is indeed to avoid that this line is logged by default, and you need to explicitly configure log4j to enabled this log, and you can choose which of the keys to log.